.infrastructure/10_dns.tf

```
@@ -107,3 +107,33 @@ resource "aws_route53_record" "devportal_google_site_verification_txt" {
   records = ["google-site-verification=Z94dFrXZD0YqP-r5BY5ODb4NsbQBAggTGRZM9fNtOj0"]
   ttl     = 3600
 }
+
+# Add DNS record for CMS Strapi
+module "records" {
```

```suggestion
module "cms_dns_records" {
```
or, if you have a better name, feel free to propose it